components/eam/src/chemistry/modal_aero/aero_model.F90

```diff
@@ -856,6 +854,50 @@ subroutine aero_model_init( pbuf2d, species_class, iflagaa )
     index_tot_mass(4,3) = get_spc_ndx('mom_a4')
     index_chm_mass(4,1) = get_spc_ndx('bc_a4' )
     !
+#elif ( ( defined MODAL_AERO_4MODE_MOM ) && ( defined RAIN_EVAP_TO_COARSE_AERO ) )
```

Comment:
Why is this new block of code needed (for the default aerosol scheme 4MODE_MOM)?

Reply:
Currently, this whole big chunk of code only matters for MAM5, as it use diagnostic diameter/surface area (SAD). If use read in SAD file (default for MAM4),  then the setting for index_tot_mass/index_chm_mass does not matter. Just put there for potential use?

Also, this 4MODE_MOM + RAIN_EVAP is missing (but it is the default for MAM4), so I add this part.